src/databricks/labs/ucx/aws/access.py

```diff
@@ -374,11 +379,15 @@ def create_uber_principal(self, prompts: Prompts):
             logger.error(f"Failed to assign instance profile to cluster policy {iam_role_name}")
             self._aws_resources.delete_instance_profile(iam_role_name, iam_role_name)
 
+    def _clean_location_name(self, location: str) -> str:
```

Comment:
Can the `clean_location` be used here too?

Reply:
That serves a different purpose. It creates a valid name for the external location. I'll rename it for clarity.